projects/libssh/build.sh

```diff
@@ -23,6 +23,21 @@ cmake -DCMAKE_C_COMPILER="$CC" -DCMAKE_CXX_COMPILER="$CXX" \
     -DBUILD_SHARED_LIBS=OFF -DWITH_INSECURE_NONE=ON $SRC/libssh
 make "-j$(nproc)"
 
+# The new fuzzers are c files
+fuzzers=$(find $SRC/libssh/tests/fuzz/ -name "*_fuzzer.c")
+for f in $fuzzers; do
+    fuzzerName=$(basename $f .c)
+    echo "Building fuzzer $fuzzerName"
+    $CC $CFLAGS -I$SRC/libssh/include/ -I$SRC/libssh/src/ -I$BUILD/ -I$BUILD/include/ \
+        "$f" -o "$OUT/$fuzzerName" -O0 -g \
+        $LIB_FUZZING_ENGINE ./src/libssh.a -Wl,-Bstatic -lcrypto -lz -Wl,-Bdynamic
+
+    if [ -d "$SRC/libssh/tests/fuzz/${fuzzerName}_corpus" ]; then
+        zip -j $OUT/${fuzzerName}_seed_corpus.zip $SRC/libssh/tests/fuzz/${fuzzerName}_corpus/*
+    fi
+done
+# This is the old building of the fuzzers in cpp files, which can be removed
+# after merging https://gitlab.com/libssh/libssh-mirror/-/merge_requests/225
```

Comment:
Let's remove this now that the PR has been merged?

Reply:
yeah. I can do that. I was not sure how long it will take to merge both of the stuff and how fresh git revisions are going to be picked by the oss-fuzz so I rather kept both initially. I can remove the cpp if it is not going to cause any troubles.